lib/builder/builder_test.go

```diff
@@ -97,6 +112,312 @@ func (s *BuilderSuite) TestSelectRuntimeVersion(c *check.C) {
 	c.Assert(err, check.ErrorMatches, "unsupported base image .*")
 }
 
+func (s *BuilderSuite) TestBuildInstallerWithDefaultPlanetPackage(c *check.C) {
+	// setup
+	remoteEnv := newEnviron(c)
+	defer remoteEnv.Close()
+	stateDir := c.MkDir()
+	appDir := c.MkDir()
+	manifestPath := filepath.Join(appDir, defaults.ManifestFileName)
+	manifestBytes := []byte(`
+apiVersion: bundle.gravitational.io/v2
+kind: Bundle
+metadata:
+  name: app
+  resourceVersion: "0.0.1"
+installer:
+  flavors:
+    items:
+      - name: "one"
+        nodes:
+          - profile: master
+            count: 1
+      - name: "three"
+        nodes:
+          - profile: master
+            count: 1
+          - profile: node
+            count: 2
+nodeProfiles:
+  - name: master
+    labels:
+      node-role.kubernetes.io/master: "true"
+  - name: node
+    labels:
+      node-role.kubernetes.io/node: "true"
+systemOptions:
+  #dependencies:
+  #  runtimePackage: gravitational.io/planet:0.0.1
+  runtime:
+    version: 0.0.1
+`)
+	writeFile(manifestPath, manifestBytes, c)
+	b, err := New(Config{
+		FieldLogger:      logrus.WithField(trace.Component, "test"),
+		Progress:         utils.DiscardProgress,
+		StateDir:         stateDir,
+		OutPath:          filepath.Join(stateDir, "app.tar"),
+		ManifestPath:     manifestPath,
+		SkipVersionCheck: true,
+		GetRepository: func(*Builder) (r string, err error) {
+			return "repository", nil
+		},
+		NewSyncer: func(*Builder) (Syncer, error) {
+			return NewPackSyncer(remoteEnv.Packages, remoteEnv.Apps, "repository"), nil
+		},
+	})
+	c.Assert(err, check.IsNil)
+
+	createRuntimeApplication(remoteEnv, c)
+	createApp(manifestBytes, remoteEnv.Apps, c)
+
+	// verify
+	err = Build(context.TODO(), b)
+	c.Assert(err, check.IsNil)
+}
+
+func (s *BuilderSuite) TestBuildInstallerWithDefaultPlanetPackageFromHub(c *check.C) {
+	// setup
+	remoteEnv := newEnviron(c)
+	defer remoteEnv.Close()
+	stateDir := c.MkDir()
+	appDir := c.MkDir()
+	manifestPath := filepath.Join(appDir, defaults.ManifestFileName)
+	manifestBytes := []byte(`
+apiVersion: bundle.gravitational.io/v2
+kind: Bundle
+metadata:
+  name: app
+  resourceVersion: "0.0.1"
+installer:
+  flavors:
+    items:
+      - name: "one"
+        nodes:
+          - profile: master
+            count: 1
+      - name: "three"
+        nodes:
+          - profile: master
+            count: 1
+          - profile: node
+            count: 2
+nodeProfiles:
+  - name: master
+    labels:
+      node-role.kubernetes.io/master: "true"
+  - name: node
+    labels:
+      node-role.kubernetes.io/node: "true"
+systemOptions:
+  #dependencies:
```

Comment:
Remove?

Reply:
I've added a couple of tests where this is used, so I'll update the branch shortly.